recipes-tag/23-id-1-csx1-collection/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set year = "2019" %}
-{% set cycle = "2" %}
-{% set version ="0" %}
+{% set cycle = "3" %}
+{% set version = "0" %}
```

Comment:
For this package the dependency of `23-id-1-csx1-analysis >=2019C2.0` should be better enforced, I think, i.e. `23-id-1-csx1-analysis >=2019C3.0`.

Reply:
I see.